index.php

```diff
@@ -30,67 +13,16 @@
             <h1>The various guinea pigs of biological science: collectable in the home or lab</h1>
         </header>
         <section class="container">
-<!--placeholders at the moment-->
-<!--                create divs with lists to float-->
-            <div class="items">
-                <ul>
-                    <li>common name</li>
-                    <li>scientific name</li>
-                    <li>kingdom</li>
-                    <li>genome size Mbp</li>
-<!--                        add background image on each div-->
-                </ul>
-            </div>
-
-            <div class="items">
-                <ul>
-                    <li>common name</li>
-                    <li>scientific name</li>
-                    <li>kingdom</li>
-                    <li>genome size Mbp</li>
-                    <!--                        add background image on each div-->
-                </ul>
-            </div>
-
-            <div class="items">
-                <ul>
-                    <li>common name</li>
-                    <li>scientific name</li>
-                    <li>kingdom</li>
-                    <li>genome size Mbp</li>
-                    <!--                        add background image on each div-->
-                </ul>
-            </div>
-
-            <div class="items">
-                <ul>
-                    <li>common name</li>
-                    <li>scientific name</li>
-                    <li>kingdom</li>
-                    <li>genome size Mbp</li>
-                    <!--                        add background image on each div-->
-                </ul>
-            </div>
-
-            <div class="items">
-                <ul>
-                    <li>common name</li>
-                    <li>scientific name</li>
-                    <li>kingdom</li>
-                    <li>genome size Mbp</li>
-                    <!--                        add background image on each div-->
-                </ul>
-            </div>
-
-            <div class="items">
-                <ul>
-                    <li>common name</li>
-                    <li>scientific name</li>
-                    <li>kingdom</li>
-                    <li>genome size Mbp</li>
-                    <!--                        add background image on each div-->
-                </ul>
-            </div>
+
+            <?php
+            require_once 'functions.php';
+
+            $db = connectDb();
+            $modelOrganisms= getDatafromDB($db);
+            $html = displayItems($modelOrganisms);
+            echo $html;
+            ?>
```

Comment:
Most of the php can go at the top of the page.
Can then use a single line here: 
```suggestion
           <?php echo $html; ?>
```

Reply:
good idea